core/src/components/item-divider/test/spec/item-divider.e2e.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('item-divider: spec', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/src/components/item-divider/test/spec');
+  });
+
+  test('should not have visual regressions', async ({ page }) => {
+    await page.setIonViewport();
+
+    expect(await page.screenshot()).toMatchSnapshot(`list-item-divider-${page.getSnapshotSettings()}.png`);
```

Comment:
Opted for `list-` here since the divider is always rendered inside of `ion-list`. We do this a few other places. 

Can rename to `item-divider-spec-` if we don't fee like that is valuable context on the image name.